docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -366,6 +366,46 @@ For example, select the top 10 percent of nations by population::
         )
       );
 
+.. _sqlserver-procedure-function:
+
+``procedure(varchar) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``procedure`` function allows you to run stored procedure on the underlying
+database directly. It requires syntax native to SQL Server.
```

Comment:
"It requires" -> "The function requires"

Reply:
I'll remove that line.